support-frontend/assets/components/subscriptionCheckouts/addressSearch/addressResultsDropdownStyles.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import { css } from '@emotion/core';
+import { textSans } from '@guardian/src-foundations/typography/obj';
+import { border } from '@guardian/src-foundations/palette';
+
+export const list = css`
+  overflow: auto;
+  max-height: 350px;
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 999;
+  background: white;
+  width: 100%;
+  border: solid 1px ${border.secondary};
+`;
+
+export const listItem = (selected: boolean) => css`
+  padding: 0px 4px;
+  background: ${selected ? '#eee' : '#fff'};
+  ${textSans.small()}
+  .highlight {
+    font-weight: bold;
+  }
+`;
```

Comment:
I'm not 100% sure of this but I have a feeling that this is the sort of dynamic css that Simon warned us against using as it's quite costly. However, I could be wrong! And it certainly is nice to be able to abstract this away from the component.